src/ray/object_manager/plasma/plasma.h

```diff
@@ -16,14 +16,14 @@
 // under the License.
 
 #pragma once
+#include <stddef.h>
```

Comment:
is it needeD?

Reply:
yeah, `ptrdiff_t` need including stddef.h.